.github/workflows/pull_request.yml

```diff
@@ -47,3 +47,45 @@ jobs:
         hal_refsi_thread_mode: WG
         debug_support: ON
         gtest_launcher: "/usr/bin/python;-u;${{ github.workspace }}/scripts/gtest-terse-runner.py"
+
+############### JOB   mr-ubuntu-clang-x86-llvm-latest-cl3-0-offline:
+
+  mr-ubuntu-clang-x86-llvm-latest-cl3-0-offline:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 90
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v4.1.0
+    - name: setup-ubuntu
+      uses: ./.github/actions/setup_ubuntu_build
+      with:
+        llvm_version: '18'
+        llvm_build_type: RelAssert
+        arch: x86
+    - run: echo WORKSPACE is $GITHUB_WORKSPACE && echo PWD is `pwd` && ls -al
+    - name: build ock one
+      uses: ./.github/actions/do_build_ock
+      with:
+        arch: x86
+        enable_api: ""
```

Comment:
I'd definitely like to review these that are likely to be defaults, it feels like this is one of them (linker etc is another).

Reply:
I think you're right on this one. I was assuming changes needed to legacy defaults and any others would become clearer as more jobs were added and we reach "critical mass" in terms of job numbers. We could review in advance if that makes more sense. For now I've updated the `enable_api` default.